documentation/optee_design.md

```diff
@@ -777,6 +778,66 @@ consists of:
 | `uint8_t[hash_size]` | hash | Hash of the fields above and the `<ELF>` above |
 | `uint8_t[sig_size]` | signature | Signature of hash |
 
+# 13. Device Tree support
+
+OP-TEE core can use the device tree format to inject platform configuration
+information during platform initialization and possibly some run time context.
+
+Device Tree technology allows to describe platforms from ASCII source files
+so-called DTS files. These can be used to generate a platform description
+binary image embedded in the platform boot media for applying expected
+configuration settings during the platform initializations.
+
+This scheme relaxes design constrains on the OP-TEE core implementation as
+most of the platform specific hardware can be tuned without modifying C
+source files or adding configuration directives in the build environments.
+
+## Secure and Non Secure Device Trees
+
+When OP-TEE is embedded into a platform, it is likely there exist a non secure
```

Comment:
s/non secure/non-secure/
here and in the rest of the file

Reply:
ok, thanks.